### PR TITLE
Leo fix visualization power

### PR DESCRIPTION
**Pull request by leoscholl**
address #149

### CODE REVIEW

File: aopy/analysis.py [hunk 1]
```diff
@@ -562,3 +562,32 @@ def find_outliers(data, std_threshold):
                   
     return good_data_idx.flatten(), distances.flatten()
 
+def calc_freq_domain_amplitude(data, samplerate, rms=False):
+    '''
+    Use FFT to decompose time series data into frequency domain to calculate the
+    amplitude of the positive frequency components
+
+    Args:
+        data (nt, nch): timeseries data in volts, can also be a single channel vector
```
Comment (rcanfield2) [inline aopy/analysis.py:571]: Doesn't necessarily need to be in volts.

File: aopy/visualization.py [hunk 2]
```diff
@@ -204,12 +216,13 @@ def plot_raster(data, plot_cue, cue_bin, ax):
        Create a raster plot of neural data
 
        Args:
-           data (n_trials, n_neurons, n_timebins): neural spiking data (not spike count- must contain only 0 or 1) in the form of a three dimensional matrix
-           plot_cue : If plot_cue is true, a vertical line showing when this event happens is plotted in the rastor plot
-           cue_bin : time bin at which an event occurs. For example: Go Cue or Leave center
+            data (n_trials, n_neurons, n_timebins): neural spiking data (not spike count- must contain only 0 or 1) in the form of a three dimensional matrix
```
Comment (rcanfield2) [inline aopy/visualization.py:219]: Can we make the first dimension time and change n_neurons to n_channels? If using multiunit activity we don't know spikes are from one neuron.

File: aopy/visualization.py [hunk 3]
```diff
@@ -204,12 +216,13 @@ def plot_raster(data, plot_cue, cue_bin, ax):
...
+            plot_cue : If plot_cue is true, a vertical line showing when this event happens is plotted in the rastor plot
+            cue_bin : time bin at which an event occurs. For example: Go Cue or Leave center
```
Comment (rcanfield2) [inline aopy/visualization.py:221]: What should we set cue_bin to if plot_cue=False?

File: aopy/visualization.py [hunk 4]
```diff
@@ -204,12 +216,13 @@ def plot_raster(data, plot_cue, cue_bin, ax):
        Create a raster plot of neural data
```
Comment (rcanfield2) [inline aopy/visualization.py:217]: Would it be possible to add an example plot?

File: tests/test_analysis.py [hunk 5]
```diff
@@ -138,6 +138,22 @@ def test_calc_rms(self):
...
+
+        # Expect 100 and 50 Hz peaks at 1 V each
+        self.assertAlmostEqual(ampls[freqs==100., 0][0], 1)
+        self.assertAlmostEqual(ampls[freqs==50., 0][0], 1)
+        self.assertAlmostEqual(ampls[freqs==25., 0][0], 0)
+
+        # Expect 1/sqrt(2) V RMS
+        freqs, ampls = aopy.analysis.calc_freq_domain_amplitude(data, samplerate, rms=True)
+        self.assertAlmostEqual(ampls[freqs==100., 0][0], 1/np.sqrt(2))
+
```
Comment (rcanfield2) [inline tests/test_analysis.py:156]: Can you run one of these tests with 2D data?

**Review by rcanfield2 — APPROVED** [review]

File: aopy/visualization.py (hunk 2 above)
Comment (leoscholl) [reply aopy/visualization.py:219]: can you move these plot_raster() comments to the issue #154

**Pull request merged**